packages/foreman/rubygem-gettext/rubygem-gettext.spec

```diff
@@ -1,181 +1,108 @@
 
-#%%{__rm} -f .%{gem_instdir}/Rakefile
-%{__rm} -f .%{gem_instdir}/%{gem_name}.gemspec
-%{__rm} -rf .%{gem_instdir}/po/
-%{__chmod} 0755 .%{gem_instdir}/bin/*
-%{__chmod} 0644 .%{gem_dir}/cache/*.gem
-find .%{gem_instdir}/ -name \*.po | xargs %{__chmod} 0644
-
-# Cleanups for rpmlint
-find .%{gem_instdir}/lib/ -name \*.rb | while read f
-do
-	%{__sed} -i -e '/^#!/d' $f
-done
-
 %install
-%{__mkdir_p} %{buildroot}{%{gem_dir},%{_bindir}}
-
-%{__cp} -a .%{_bindir}/* %{buildroot}/%{_bindir}/
-%{__cp} -a .%{gem_dir}/* %{buildroot}%{gem_dir}/
-find %{buildroot}%{gem_dir} -name \*.rb.patch\* -delete
-
-
-# For --short-circult
-%{__rm} -f *.lang
-
-# modify find-lang.sh to deal with gettext .mo files under
-# %%{gem_instdir}/locale (required on EL6)
-sed -e 's|/share/locale/|/locale/|' \
-	/usr/lib/rpm/find-lang.sh \
-	> find-lang-modified.sh
+mkdir -p %{buildroot}%{gem_dir}
+cp -a .%{gem_dir}/* \
+        %{buildroot}%{gem_dir}/
 
-sh find-lang-modified.sh %{buildroot} gettext gettext-gem.lang
-#%find_lang gettext
-#mv gettext.lang gettext-gem.lang
+mkdir -p %{buildroot}%{_bindir}
+cp -a .%{_bindir}/* \
+        %{buildroot}%{_bindir}/
+find %{buildroot}%{gem_instdir}/bin -type f | xargs chmod a+x
 
-%{__cat} *-gem.lang >> %{pkg_name}-gem.lang
-
-# list directories under %%{gem_instdir}/locale/
-find %{buildroot}%{gem_instdir}/locale -type d | while read dir
-do
-	echo "%%dir ${dir#%{buildroot}}" >> %{pkg_name}-gem.lang
-done
-
-# replace shebangs to prevent SCL packages depending on non-SCL Ruby
-find %{buildroot}%{gem_instdir}/samples/ -name *.rb -exec \
-  sed -ri '1sX/usr/bin/rubyX/usr/bin/env rubyX' {} +
-
-# clean up
-rm -f %{buildroot}%{gem_instdir}/.yardopts
-
-# Disable tests as it adds test-unit requirement into builds, and currently
-# this isn't actively used.
-#%check
-#pushd .%{gem_instdir}
-#export LANG=ja_JP.UTF-8
-#%{?scl:scl enable %{scl} "}
-#ruby -Ilib:test test/run-test.rb
-#%{?scl:"}
-#popd
-
-
-%files	-f %{pkg_name}-gem.lang
-%defattr(-,root,root,-)
-%{_bindir}/rxgettext
-%{_bindir}/rmsginit
+%files
+%dir %{gem_instdir}
 %{_bindir}/rmsgcat
-%{_bindir}/rmsgfmt
 %{_bindir}/rmsgmerge
-
-%dir %{gem_instdir}/
-%doc %{gem_instdir}/[A-Z]*
-%doc %{gem_instdir}/doc/
-%exclude %{gem_instdir}/Rakefile
-%{gem_instdir}/bin/
-%{gem_instdir}/lib/
-
-%exclude	%{gem_cache}
+%{_bindir}/rxgettext
+%{_bindir}/rmsgfmt
+%{_bindir}/rmsginit
+%exclude %{gem_instdir}/.yardopts
+%{gem_instdir}/bin
+%{gem_libdir}
+%{gem_instdir}/locale
+%{gem_instdir}/po
+%{gem_instdir}/samples
+%{gem_instdir}/src
+%exclude %{gem_cache}
 %{gem_spec}
 
-%files		doc
-%defattr(-,root,root,-)
-%{gem_docdir}/
-%{gem_instdir}/samples/
-%exclude	%{gem_instdir}/test/
-%exclude	%{gem_instdir}/src/
+%files doc
+%doc %{gem_docdir}
+%doc %{gem_instdir}/README.md
+%{gem_instdir}/Rakefile
+%doc %{gem_instdir}/doc
+%{gem_instdir}/gettext.gemspec
+%{gem_instdir}/test
 
 %changelog
+* Wed Apr 08 2020 Zach Huntington-Meath <zhunting@redhat.com> - 3.1.4-9
+- Bump release to build for el8
```

Comment:
This is more than just a release bump

Reply:
Ack, sorry this was before I had revamped it.